staging/src/k8s.io/apiserver/pkg/features/kube_features.go

```diff
@@ -34,15 +34,6 @@ const (
 	// redirects from the backend (Kubelet) for streaming requests (exec/attach/port-forward).
 	StreamingProxyRedirects utilfeature.Feature = "StreamingProxyRedirects"
 
-	// owner: @tallclair
-	// alpha: v1.7
-	// beta: v1.8
-	//
-	// AdvancedAuditing enables a much more general API auditing pipeline, which includes support for
-	// pluggable output backends and an audit policy specifying how different requests should be
-	// audited.
-	AdvancedAuditing utilfeature.Feature = "AdvancedAuditing"
```

Comment:
do we have a deprecation policy of feature gates? Will kube-apiserver refuse to start with `AdvancedAuditing=true` from now on?

Reply:
I haven't found any recommendations for deprecation of feature gates, so I assume common approach should be used. Considering that it was not mentioned anywhere that feature gate is deprecated, we can't just remove it from the code (I'll fix it in my PR).

I don't see any sense in keeping this feature gate functional, considering that it was used only for enabling legacy audit logging. So my proposal is to stop depending on it (advanced audit logging is not enabled if policy file is not specified), deprecate and remove in later releases (not sure if we should wait for 1 or 2 releases in case of Beta).